src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -1515,6 +1519,20 @@ fn range_between_backticks(ori_link: &MarkdownLink) -> Range<usize> {
         ..(ori_link.range.start + before_second_backtick_group)
 }
 
+/// Returns true if we should ignore `path_str` due to it being unlikely
+/// that it is an intra-doc link.
+/// The difference between this and [`should_ignore_link()`] is that this
+/// check should only be used on links that still have disambiguators.
```

Comment:
```suggestion
/// Returns true if we should ignore `path_str` due to it being unlikely
/// that it is an intra-doc link.
///
/// The difference between this and [`should_ignore_link()`] is that this
/// check should only be used on links that still have disambiguators.
```

Reply:
I intentionally put it all in the same paragraph so that the difference sentence would show up in the summary. But I can split them if you like.